### DISCUSSION TITLE
Export to Freetube

### DISCUSSION THREAD

**Discussion by chris02000** [InfinityLoop1308/PipePipe#404] [Q&A] [answered]
Hi, Great app, thanks.
I would like to export to freetube for my PC is it possible?
Even better I would like to sync the two.

Thanks
Chris

**Comment by InfinityLoop1308** [comment, marked as answer]
Hi Chris,

Unfortunately, PipePipe's backups are unlikely to be compatible with FreeTube because they have completely different designs. Even if someone creates a third-party conversion tool, most of the information would be lost during the conversion.
